src/utils/components/AddBootableVolumeModal/components/VolumeDestination/VolumeDestination.tsx

```diff
@@ -34,6 +45,14 @@ const VolumeDestination: FC<VolumeDestinationProps> = ({
             storageClass={storageClassName}
           />
         </GridItem>
+        <GridItem span={12}>
+          <ApplyStorageProfileSettingsCheckbox
+            claimPropertySets={claimPropertySets}
+            disabled={!storageProfileLoaded}
+            handleChange={handleApplyOptimizedSettingsChange}
```

Comment:
I think handleChange={setApplyStorageProfile} should work

Reply:
Done!